### ISSUE TITLE
Possibly need clarification on Nuxt fetch in "Fetching content" page

### ISSUE THREAD

**Issue by azrikahar** [nuxt/content#445]
In the first paragraph of the [Fetching content](https://content.nuxtjs.org/fetching) page of the docs, it mentioned the fetch method being able to access Nuxt Content via `context.$content`, quoted below:

>  This module globally injects `$content` instance, meaning that you can access it anywhere using `this.$content`. For plugins, asyncData, **fetch**, nuxtServerInit and Middleware, **you can access it from `context.$content`**.

This statement is only true for Nuxt <= 2.11 with the old fetch. It might be confusing for readers that are more familiar with the new fetch method only works with `this.$content` instead of `context.$content`.

Maybe a warning box to describing the difference between versions would suffice? or should we just reword it to encourage readers to use the new fetch method and don't mention the old fetch can access it via context?

**Comment by mathe42** [comment]
In `fetch` you get the context as first argument so you can use it with `context.$content` but you can also use `this.$content`!

**Comment by benjamincanac** [comment]
Hey @azrikahar, nice catch, indeed this was written before the new fetch, I just removed it from the list.

In my opinion it implicitly says that fetch is accessible from `this`.

What do you think?

**Comment by azrikahar** [comment]
@benjamincanac I agree. Let's keep it simple for now :)

Just being the overthinking person that I am ( and a tendency to over-explain stuffs :/ ), i think if there is ever a need to clarify this implicit meaning in the future, I suppose we can add a `fetch()` snippet in the [Nuxt Content docs Snippets page](https://content.nuxtjs.org/snippets) since there is one for `asyncData()` already.

**Comment by benjamincanac** [comment]
@azrikahar Indeed it would be really nice 🙂
